src/LoginWithAmazon.test.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TestUtils from 'react';
+import { shallow, mount } from 'enzyme';
+import LoginWithAmazon from './LoginWithAmazon';
+
+let loginWithAmazon;
+let loginWithAmazonInstance;
+let mockSetAccessToken;
+let authorize;
+
+beforeEach(() => {
+    mockSetAccessToken = jest.fn();
+    loginWithAmazon = shallow(<LoginWithAmazon setAccessToken={mockSetAccessToken}/>);
+    loginWithAmazonInstance = loginWithAmazon.instance();
+    authorize = jest.fn();
+
+    Object.defineProperty(window, 'amazon', {
+        value: { Login: { authorize: authorize } },
+        configurable: true
+    });
+});
+
+it('renders without crashing', () => {
```

Comment:
This isn't testing render without crash. Write a separate test for that and change the description here.